src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist-groovy2

```diff
@@ -6,6 +6,15 @@ staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods each java.util.Lis
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods each java.util.Map groovy.lang.Closure
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods each java.util.Set groovy.lang.Closure
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods each java.util.SortedSet groovy.lang.Closure
+
+staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods inject java.lang.Iterable java.lang.Object groovy.lang.Closure
+staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods inject java.util.Collection java.lang.Object groovy.lang.Closure
+staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods inject java.util.Iterator java.lang.Object groovy.lang.Closure
```

Comment:
At least this overload, and probably others, is present in Groovy 1.x and thus should be listed in `generic-whitelist`, not here.